akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -671,4 +671,20 @@ object Source {
   def unfoldResourceAsync[T, S](create: () ⇒ Future[S], read: (S) ⇒ Future[Option[T]], close: (S) ⇒ Future[Done]): Source[T, NotUsed] =
     Source.fromGraph(new UnfoldResourceSourceAsync(create, read, close))
 
+  /**
+   * Materializes into a sink connected to a source, i.e. the sink pushes into the source:
+   *
+   * +----------+       +----------+
+   * >   Sink   |------>|  Source  >
+   * +----------+       +----------+
+   *
+   * Should be provided by Akka Streams, see https://github.com/akka/akka/issues/24853.
+   */
+  def sinkToSource[M]: RunnableGraph[(Sink[M, NotUsed], Source[M, NotUsed])] =
```

Comment:
`M` is typically used for the type of the materialized value. Please change to `A` or `T`.